docs/metamodel.md

```diff
@@ -318,3 +318,27 @@ assignments (`=`). An optional assignment (`?=`) will always be `False` if the
 RHS object is not matched in the input. The multiplicity assignments (`*=` and
 `+=`) will always be python lists.
 
+## Optional model parameter definitions
+
+A metamodel can define optional model parameter definitions. Such definitions
+are stored in `_tx_model_param_definitions` and define optional parameters,
+which can be specified while loading/creating a model through `model_from_str`
+or `model_from_file`. Details: see [tx_model_params](model.md#_tx_model_params).
+
+`metamodel._tx_model_param_definitions` can be queried (like a dict) to
+retrieve possible extra parameters and their description for a metamodel.
+It is also used to restrict the additional parameters passed to
+`model_from_str` or `model_from_file`.
+
+Default parameters are:
+
+ * `project_root`: This parameter defines the root of the project and is used by
```

Comment:
...defines the root path of the project...

Reply:
done